docs/schemas/v1/definitions.json

```diff
@@ -20,6 +20,17 @@
             "authorization": {
               "type": "object",
               "properties": {
+                "providers": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
```

Comment:
This should probably be an object, so that we can add configuration to it. I'd also make `authentication` separate from `authorization`